.github/workflows/build.yml

```diff
@@ -138,68 +138,29 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
   linux-arm64:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-22.04-arm
     strategy:
       matrix:
         build_type: [ Debug, Release ]
       fail-fast: false
     steps:
       - uses: actions/checkout@v2
-      - uses: uraimo/run-on-arch-action@v2.8.1
+      - uses: addnab/docker-run-action@v3
```

Comment:
Is docker still necessary?

Reply:
I guess it's necessary because we want to get an older Ubuntu version?

Reply:
@eymar unfortunately yes - because we need to build in Ubuntu 20.04 - and the minimal arm machine in github runner is Ubuntu 22.04